src/Tasks/Common/NETSdkMessage.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Framework;
+using System;
+
+namespace Microsoft.NET.Build.Tasks
+{
+    /// <summary>
+    /// Provides a localizable mechanism for logging messages with different levels of importance from the SDK targets.
+    /// </summary>
+    public class NETSdkMessage : MessageBase
```

Comment:
This one should not be in common. Each task library should have a unique name for any tasks.

Reply:
I will move it.